### ISSUE TITLE
No 1.7 McXtrace version for mac

### ISSUE THREAD

**Issue by antoinepado** [McStasMcXtrace/McCode#1235]
There is no 1.7 McXtrace version for mac [here](https://download.mcxtrace.org/mcxtrace-1.7/mac/mcxtrace-mac_x86_64/)

**Comment by willend** [comment]
This is true - and there is another issue with part of an explanation here: https://github.com/McStasMcXtrace/McCode/issues/1229

I will build one asap

**Comment by willend** [comment]
Should be there now - build date is today of course.

**Comment by willend** [comment]
Sorry, removed again.

I will do a rebuild against current master instead of the mcxtrace-1.7 release branch. Tobias Weber has made a relevant correction of the environment script...

**Comment by willend** [comment]
The installer is now in place. I have built against mcxtrace-1.7-release, cherry-picking 651c3ca0422870d27464029e79b19599dacd9588

**Comment by willend** [comment]
I have tested it here, please install where it was needed @antoinepado and let me know how things go. If succesfull we may close this issue and #1229
